### PR TITLE
MDE/KBE-57

### PR DESCRIPTION
**Pull request by mantvydasdeltuva**
# Initial (2024-12-21)
- [MDE/KBE-57 Implemented clinvar helpers](https://github.com/mantvydasdeltuva/kath/commit/8f5387ca39d48b8179bb6f447278a87bbb2073e1)
- [MDE/KBE-57 Implemented clinvar eys gene data download](https://github.com/mantvydasdeltuva/kath/commit/0f13e52a1dfb836b1245249e8f1fef45937c7183)
- [MDE/KBE-57 Linked clinvar download functionality to system](https://github.com/mantvydasdeltuva/kath/commit/4cd435f8b0b52c2e15feba360b9e3dec678c912c)
# Changes (2024-12-22)
- [MDE/KBE-57 Refactored " and ' chars](https://github.com/mantvydasdeltuva/kath/pull/10/commits/de549dc1c0fd78d071c4e3dd4d8f55becbe5283f)

### CODE REVIEW

File: app/back_end/src/data/refactoring.py [hunk 1]
```diff
@@ -347,3 +349,107 @@ def find_popmax_in_gnomad(data:pd.DataFrame):
...
+
+        # AlleleID(s)
+        simple_allele = element.find("ClassifiedRecord/SimpleAllele")
+        allele_id = simple_allele.attrib.get("AlleleID") if simple_allele is not None else None
+        row.append(allele_id if allele_id is not None else "")
+
+        # dbSNP ID
+        xrefs = element.findall('ClassifiedRecord/SimpleAllele/XRefList/XRef')
+        xref = next((inner for inner in xrefs if inner.attrib.get("DB") == "dbSNP"), None)
+        row.append(f"{xref.attrib.get("Type")}{xref.attrib.get("ID")}" if xref is not None else "")
```
Comment (justinnas) [inline app/back_end/src/data/refactoring.py]: "Type" needs to be in single quotes ('Type'), otherwise it causes errors

**Review by justinnas — CHANGES_REQUESTED** [review]
Everything looks good apart one small error

File: app/back_end/src/data/refactoring.py (hunk 1 above)
Comment (mantvydasdeltuva) [reply app/back_end/src/data/refactoring.py]: Resolved with [MDE/KBE-57 Refactored " and ' chars](https://github.com/mantvydasdeltuva/kath/pull/10/commits/de549dc1c0fd78d071c4e3dd4d8f55becbe5283f)

**Review by justinnas — APPROVED** [review]
Well done!!

**Pull request merged**